saleor/graphql/account/mutations/service_account.py

```diff
@@ -63,8 +79,19 @@ class Meta:
         description = "Deletes an authentication token assigned to service account."
         model = models.ServiceAccountToken
         permissions = (AccountPermissions.MANAGE_SERVICE_ACCOUNTS,)
-        error_type_class = AccountError
-        error_type_field = "account_errors"
+        error_type_class = ServiceAccountError
+        error_type_field = "service_account_errors"
+
+    @classmethod
+    def clean_instance(cls, info, instance):
+        requestor = info.context.user
```

Comment:
Do we limit/block mutation for Create/Update/Delete Token to be available only for `user`? If not what about scenario when `ServiceAccount` (with perm `manage_service_accounts`)  will call this mutation? Did you test it with `AnonymousUser` and assigned `ServiceAccount`?

Reply:
I made it available also for service accounts.